services/skus/handler/handler.go

```diff
@@ -67,3 +77,55 @@ func (h *Order) Create(w http.ResponseWriter, r *http.Request) *handlers.AppErro
 
 	return handlers.RenderContent(ctx, order, w, http.StatusCreated)
 }
+
+func (h *Order) CreateNew(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+	raw, err := io.ReadAll(io.LimitReader(r.Body, reqBodyLimit10MB))
+	if err != nil {
+		return handlers.WrapError(err, "Failed to read request body", http.StatusBadRequest)
+	}
+
+	req := &model.CreateOrderRequestNew{}
+	if err := json.Unmarshal(raw, req); err != nil {
+		return handlers.WrapError(err, "Failed to deserialise request", http.StatusBadRequest)
+	}
+
+	ctx := r.Context()
+
+	if err := h.valid.StructCtx(ctx, req); err != nil {
+		verrs, ok := collectValidationErrors(err)
+		if !ok {
+			return handlers.WrapError(err, "Faled to validate request", http.StatusBadRequest)
```

Comment:
typo failed

Reply:
Sorry about that. I re-read my last PR for typos, but here they are still present. Will pay more attention to it.

Reply:
Fixed, thanks!